READMEs/datatokens-flow.md

```diff
@@ -68,14 +81,43 @@ from ocean_lib.web3_internal.wallet import Wallet
 private_key = os.getenv('TEST_PRIVATE_KEY1')
 config = ExampleConfig.get_config()
 ocean = Ocean(config)
+print(f"config.network_url = '{config.network_url}'")
+print(f"config.block_confirmations = {config.block_confirmations.value}")
 
-print("create wallet: begin")
+print("Create wallet: begin")
 wallet = Wallet(ocean.web3, private_key, config.block_confirmations, config.transaction_timeout)
-print(f"create wallet: done. Its address is {wallet.address}")
-
-print("create datatoken: begin.")
-datatoken = ocean.create_data_token("Dataset name", "dtsymbol", from_wallet=wallet)
-print(f"created datatoken: done. Its address is {datatoken.address}")
+print(f"Create wallet: done. Its address is {wallet.address}")
+
+print("Create ERC721 token: begin.")
+erc721_token = ocean.create_nft_token(name="Dataset name", symbol="dtsymbol", from_wallet=wallet)
+print(f"Created ERC721 token: done. Its address is {erc721_token.address}")
+print(f"ERC721 token name: {erc721_token.token_name()}")
+print(f"ERC721 token symbol: {erc721_token.symbol()}")
+
+# Create ERC20 token related to the above NFT.
+from ocean_lib.models.models_structures import ErcCreateData
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+
+print("Create ERC20 token: begin.")
```

Comment:
Suggested tweak: "Create ERC20 token" --> "Create ERC20 datatoken". 

And for everything below: "ERC20 token" --> "datatoken"

Reply:
Sorry, Trent, it seems you reviewed just as we were merging this. We'll handle the text changes separately